### ISSUE TITLE
Replicating benchmark NER results with Ontonotes 5

### ISSUE THREAD

**Issue by juand-r** [explosion/spaCy#2824]
# On replicating benchmark NER results with Ontonotes 5

I would like to replicate the results of the spaCy pre-trained NER model on the English portion of the Ontonotes 5 corpus (the spaCy documentation says I should get ~ 85% F1), but I could not find any details on how to replicate it exactly.

1. Which test set was used for the evaluation? I followed the instructions at http://cemantix.org/data/ontonotes.html, but there are two different test sets ("Test" and "CoNLL-2012 Test"). Also, were all subcorpora used, or was the pivot text (subdirectory `pt`, consisting of text from Old and New Testaments) removed? (since this does not have entity labels)

2. I am currently extracting entities one sentence at a time (e.g. doc = nlp(sentence_text); ents = doc.ents). Is this the right way to use spaCy's NER, or am I missing something? Doing it this way, I'm getting about 74% micro-F1 (using spaCy 2.0.12, with model en-core-web-lg 2.0.0).

Thanks!

## Which page or section is this issue related to?
https://spacy.io/usage/facts-figures#ner-accuracy-ontonotes5

**Comment by juand-r** [comment]
Thanks!  I'll see if I can reproduce the results with the development set.

**Comment by honnibal** [comment]
No worries.

If problems persist and you already have a copy of the data, I could send you our pre-processed files, which would let you reproduce it exactly.

**Comment by juand-r** [comment]
That would be very helpful. Thanks!  I already have a copy of OntoNotes 5.

I had a bug in my processing script that caused most of the discrepancy. With that fixed I'm getting  ~81.5% on the test set and ~83.5% on the development set (the difference between evaluating a sentence at a time vs a document at a time was small, less than .2%).

I wonder if the remaining 2% (to get from 83.5 to 85.81) is due to dataset selection issues or to tokenization issues (OntoNotes 5 is pre-tokenized, so I used a custom spacy tokenizer to only split tokens at whitespace).
